introduction.md

```diff
@@ -1 +1,8 @@
 Greetings, Jay here - favorite path so far has been javascript.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?